dask/dataframe/core.py

```diff
@@ -810,6 +810,34 @@ def min(self, axis=None, skipna=True):
                                  skipna=skipna, axis=axis,
                                  meta=meta)
 
+    def idxmaxmin(self, fn, axis=None, skipna=True):
+        if fn == 'idxmax':
+            idxfn = _idxmax
+        elif fn == 'idxmin':
+            idxfn = _idxmin
+        else:
+            raise ValueError('Function must be idxmax or idxmin')
+
+        axis = self._validate_axis(axis)
+        # if axis == 1:
+            # return map_partitions(idxfn, None, self,
+                                  # token=self._token_prefix + fn,
+                                  # skipna=skipna, axis=axis)
+        # else:
```

Comment:
axis=1 operates on the rows, unless validate_axis does something different. If the rows are chunked as well then we need to do the same tracking of the max/min values and recombine.  If not, then we can just call map_partitions?